content/docs/apps/frameworks.md

```diff
@@ -17,13 +17,22 @@ cloud.gov uses [buildpacks]({{< relref "docs/getting-started/concepts.md#buildpa
 
 cloud.gov supports applications written in Go, Java, Node.js, .NET Core, PHP, Python, and Ruby. cloud.gov also supports applications that rely on a static binary that uses the 64-bit Linux kernel ABI, or that consist of static HTML, CSS, and Javascript assets. See the [Cloud Foundry system (supported) buildpacks list](http://docs.cloudfoundry.org/buildpacks/#system-buildpacks) for details.
 
-## Examples
+## Generic Examples
 
 You can deploy example applications in many languages and frameworks:
 
-* [Hello worlds](https://github.com/18F/cf-hello-worlds): Java, Clojure, NodeJS, PHP, Flask (Python), Padrino (Ruby), Sinatra (Ruby)
-* [Drupal](https://github.com/18F/cf-ex-drupal)
-* [Cloud Foundry community sample applications](https://github.com/cloudfoundry-samples)
+* [Hello worlds](https://github.com/18F/cf-hello-worlds): This repository includes CloudFoundry apps in several frameworks that only have one page displaying "_Hello World from &lt;language&gt;_". Included are: Java, Clojure, .NET Core, NodeJS, PHP, Flask (Python), Padrino (Ruby), and Sinatra (Ruby).  They provide the simplest possible example from which you can build.
+* [Drupal](https://github.com/18F/cf-ex-drupal): Running Drupal on cloud.gov is fully supported. We provide the backend databases (MySQL or Postgres) and asset storage (AWS S3 buckets) so Drupal is _cloud-ready_. This repository includes examples in Drupal 7 and Drupal 8. The [Drupal 8 example](https://github.com/18F/cf-ex-drupal/tree/master/drupal-8) demonstrates use of [Composer](https://getcomposer.org/) for development, and includes S3 integration.
+* [Cloud Foundry community sample applications](https://github.com/cloudfoundry-samples): This GitHub organization has seventy examples (and counting) of languages, frameworks and tools that can be adapted to run on cloud.gov.
+
+## Customer Examples
+
+Several cloud.gov customers have their code available as open-source, including:
+
+* [College Scorecard Data API](https://github.com/18F/open-data-maker) A ruby application with an Elasticsearch backend
```

Comment:
Both this and the next item link to `https://github.com/18F/open-data-maker` but I'm not sure that's the right URL for either...